gslib/utils/copy_helper.py

```diff
@@ -3591,7 +3591,7 @@ def _ParseManifest(self):
     """
     try:
       if os.path.exists(self.manifest_path):
-        with io.open(self.manifest_path, 'r', encoding=UTF8) as f:
+        with open(self.manifest_path, 'r') as f:
```

Comment:
Please add a comment explaining why we don't use `io.open()` here to open the file in text/unicode mode in both Python versions (i.e. Python2's `csv.reader()` requires the file be opened in binary mode).

Reply:
Sure thing!